pkg/sources/github/receive_adapter/receive_adapter.go

```diff
@@ -98,30 +119,31 @@ func main() {
 	}
 }
 
-func postMessage(target string, m *github.PullRequestPayload) error {
-	jsonStr, err := json.Marshal(m)
-	if err != nil {
-		log.Printf("Error: Failed to marshal the message: %+v : %v", m, err)
-		return err
-	}
-
+func postMessage(target string, m *gh.PullRequestPayload, source, eventType, eventID string) error {
```

Comment:
If you leave `m` as an interface{}, can you avoid the cast in 59 (which could panic), and support multiple payload types idiomatically? (This may not work, I haven't tested.)

Reply:
this is good.